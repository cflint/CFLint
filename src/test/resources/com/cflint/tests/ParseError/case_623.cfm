<cfscript>
    function doSomething(id){
		switch (id){
			case -1:
				return "new";
			default:
				return "existing";
		}
    }
</cfscript>