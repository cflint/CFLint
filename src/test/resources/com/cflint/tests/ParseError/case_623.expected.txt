{
  "version" : "",
  "timestamp" : 1574137774,
  "issues" : [ ],
  "counts" : {
    "totalFiles" : 0,
    "totalLines" : 0,
    "countByCode" : [ ],
    "countBySeverity" : [ ]
  }
}